packages/theme-sections/theme-sections.js

```diff
@@ -212,13 +212,16 @@ function normalizeContainers(containers) {
 }
 
 if (window.Shopify.designMode) {
-  document.addEventListener('shopify:section:load', function(event) {
-    var id = event.detail.sectionId;
-    var container = event.target.querySelector(
-      '[' + SECTION_ID_ATTR + '="' + id + '}"]'
+  document.addEventListener('shopify:section:load', event => {
```

Comment:
Avoiding ES6 syntax to support IE11 without using Babel which keeps filesize to a minimum. Please use `function() { ... }` not arrow functions.